bigsea-controller/service/api/controller/controller_builder.py

```diff
@@ -27,6 +27,7 @@
 from service.api.controller.plugins.proportional_derivative.proportional_derivative_controller import (
```

Comment:
It looks better if you change this line to:
**from service.api.controller.plugins.proportional_derivative.controller import (**

Reply:
I would prefer to keep the classes names as they are now.

Reply:
OK, but I will keep the classes names as they are now.